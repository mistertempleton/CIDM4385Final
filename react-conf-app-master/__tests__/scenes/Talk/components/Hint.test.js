// @flow
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Hint from '../../../../app/scenes/Talk/components/Hint';

describe('Talk - Hint', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Hint onClose={() => {}} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
