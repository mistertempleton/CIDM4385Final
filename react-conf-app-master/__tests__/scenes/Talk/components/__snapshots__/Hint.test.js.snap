exports[`Talk - Hint renders correctly 1`] = `
<BVLinearGradient
  colors={
    Array [
      452261108,
      4294243572,
    ]
  }
  end={undefined}
  locations={
    Array [
      0,
      0.66,
    ]
  }
  pointerEvents="none"
  start={undefined}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "transparent",
      "bottom": 0,
      "height": 80,
      "justifyContent": "flex-end",
      "left": 0,
      "opacity": 1,
      "paddingBottom": 10,
      "position": "absolute",
      "width": 750,
    }
  }>
  <View
    style={
      Object {
        "opacity": 0.2,
        "transform": Array [
          Object {
            "translateY": 0,
          },
        ],
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#383838",
            "fontSize": 20,
          },
          undefined,
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
        ]
      }>
      
    </Text>
  </View>
</BVLinearGradient>
`;
